charts/prometheus/templates/alertmanager/pvc.yaml

```diff
@@ -25,6 +25,7 @@ spec:
 {{- if .Values.alertmanager.persistentVolume.volumeBindingMode }}
   volumeBindingModeName: "{{ .Values.alertmanager.persistentVolume.volumeBindingMode }}"
 {{- end }}
+  allowVolumeExpansion: {{ .Values.alertmanager.persistentVolume.allowVolumeExpansion | default "false" | quote }}
```

Comment:
@bseenu I believe this is down to the Storage Class config. Correct me please if I'm mistaken.

Reply:
@timurista @Xtigyro Yes this is storage class config, but in the values this is exposed under persistentVolume, please check this https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/values.yaml#L213

Reply:
@timurista @Xtigyro sorry my bad i got confused with storage class available in pvc template, yes this should be a seperate template for storage class